### DISCUSSION TITLE
Keyboard layout to abnt-2

### DISCUSSION THREAD

**Discussion by PedroCordeiroF** [prasanthrangan/hyprdots#1118] [Q&A]
I am a complete beginner, so i am having a hard time trying to change the keyboard layout to abnt2... Does someone have a step-by-step for dummies on how to change it?

**Comment by kRHYME7** [comment]
Hyprland wiki should be sufficient. 
https://wiki.hyprland.org/Configuring/Uncommon-tips--tricks/#switchable-keyboard-layouts
https://wiki.hyprland.org/Configuring/Variables/#input

edit  ``` ~/.config/hypr/hyprland.conf ``` 

find the line 

`` kb_layout = us ``` 

now add your prefered kb_layout

example: 
`` kb_layout = us,pl ``` 

dunno  if  ```  abnt2 ``` is valid so read the wiki first on how to get the kb_layout.

**Comment by DiegoSof2** [comment]
> I am a complete beginner, so i am having a hard time trying to change the keyboard layout to abnt2... Does someone have a step-by-step for dummies on how to change it?



> I am a complete beginner, so i am having a hard time trying to change the keyboard layout to abnt2... Does someone have a step-by-step for dummies on how to change it?

change it to br, in the file that kRHYME7's mentioned in the answer above, but every time you update hyprdots you will lose this configuration so also put it in the userprefs.conf file which is in the same folder
